lib/wit/gitrepo.py

```diff
@@ -214,11 +227,12 @@ def _git_command(self, *args):
 
     def _git_check(self, proc):
         if proc.returncode:
-            log.error("Command [{}] exited with non-zero exit status [{}]"
-                      .format(' '.join(proc.args), proc.returncode))
-            log.error("stdout: [{}]".format(proc.stdout.rstrip()))
-            log.error("stderr: [{}]".format(proc.stderr.rstrip()))
-            raise GitError(proc.stderr.rstrip())
+            msg = "Command [{}] exited with non-zero exit status [{}]\n".format(
+                  ' '.join(proc.args), proc.returncode)
+            msg += "stdout: [{}]\n".format(proc.stdout.rstrip())
+            msg += "stderr: [{}]\n".format(proc.stderr.rstrip())
+            msg += proc.stderr.rstrip()
```

Comment:
This adds stderr twice

Reply:
Fixed by https://github.com/sifive/wit/pull/96/commits/5acee8507ab3db97dc9ee6bab1a44b940c73ee0d